extensions/kubectl/command.go

```diff
@@ -18,17 +19,20 @@ import (
 
 const volumeName = "config"
 
-// Apply is a helper function that call a helper to create a Job and executing kubectl apply by copying the yaml
-// in the pod. It returns the output from the pod logs.
-func Apply(client *rancher.Client, cluster *apisV1.Cluster, yamlContent *management.ImportClusterYamlInput, clusterID string) (string, error) {
+// Command creates a kubernetes job to execute the provided commands.
+// It initializes necessary volume and volume mounts for the job and overrides securityContext to execute the commands.
+// If yamlContent is provided, an init container is created to write this content to a file.
+// After creating the job it waits for the execution of the job and retrieves the logs of the executed commands
+func Command(client *rancher.Client, cluster *apisV1.Cluster, yamlContent *management.ImportClusterYamlInput, clusterID string, command []string) (string, error) {
```

Comment:
Should we remove explain.go? because with command.go we could use it for both apply and explain if I understand it correctly?

Reply:
Removed, good catch! 👍